src/classes/providers/shifter/types.ts

```diff
@@ -255,8 +253,8 @@ interface IShifterChangeIp {
 
 type TShifterCountryPortMapping = Partial<Record<EShifterCountry, number[]>>;
 
-type TShifterStrategyChangeIpEveryRequest = {
-  mode: EStrategyMode.CHANGE_IP_EVERY_REQUESTS;
+type TShifterStrategy = {
+  mode?: EStrategyMode.CHANGE_IP_EVERY_REQUESTS | EStrategyMode.MANUAL;
```

Comment:
I changed it to optional and had a default strategy. I did that because I was trying to find the easiest way to use the shifter's integration from a user's perspective. No need to enforce a strategy, and you can straight up use it once you gave it some mapping.
Would you rather see it enforced? I wouldn't mind it, it's all right.

Reply:
I do have to change the README though